Rego/DefenderConfig.rego

```diff
@@ -662,20 +723,34 @@ tests[{
 # Step 4: ensure that some user is notified in the event of a DLP violation
 SensitiveRulesNotNotifying[Rule.Name] {
     Rule := SensitiveRules[_]
+    count(SensitiveInfoPolicies) > 0
+    Rule.ParentPolicyName in SensitiveInfoPolicies
     count(Rule.NotifyUser) == 0
 }
 
+DefenderErrorMessage44(Rules) := GenerateArrayString(Rules, ErrorMessage) if {
```

Comment:
Same Nitpick: This is not the 44th Defender Error message, this is the Error Message for Defender 4.4; Therefore, `DefenderErrorMessage4_4`